c2rust-transpile/src/build_files/Cargo.toml.hbs

```diff
@@ -32,7 +32,7 @@ path = "{{path}}"
 name = "{{name}}"
 {{/each}}
 [dependencies]
-{{#each dependencies~}}
+{{#each dependencies}}
```

Comment:
Doesn't this add an extra empty line between the dependencies? Afaicr that's why we had the `~`.